backend/sXb-service/Controllers/UsersController.cs

```diff
@@ -87,7 +87,7 @@ public class UsersController : Controller {
                 return BadRequest ();
 
             // Validate: .edu email address.
-            if (!Regex.Match (newUser.Email, ".+@.+[.]edu").Success) {
+            if (!Regex.Match (newUser.Email, ".+@.+[.](edu|com)").Success) {
```

Comment:
We want any top level domain. I think a '\w*' replacing '(edu|com)' will do the trick. Could be wrong though.